.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,29 @@
+settings:
+  # Jira project key to create the issue in
+  jira_project_key: "WF"
+
+  # Dictionary mapping GitHub issue status to Jira issue status
+  status_mapping:
+    opened: Untriaged
+    closed: Done
+
+  # (Optional) GitHub labels. Only issues with one of those labels will be synchronized.
+  # If not specified, all issues will be synchronized
+  labels:
+    - bug
+    - enhancement
+
+  # (Optional) (Default: false) Add a new comment in GitHub with a link to Jira created issue
+  add_gh_comment: true
+
+  # (Optional) (Default: true) Synchronize issue description from GitHub to Jira
+  sync_description: true
+
+  # (Optional) (Default: true) Synchronize comments from GitHub to Jira
```

Comment:
missing `epic_key` (`WF-63`) to ensure that the report issue gets assigned the `Github Issues` epic in jira